src/Init/Data/BitVec/Lemmas.lean

```diff
@@ -3002,6 +3002,11 @@ theorem toInt_neg_of_ne_intMin {x : BitVec w} (rs : x ≠ intMin w) :
   have := @Nat.two_pow_pred_mul_two w (by omega)
   split <;> split <;> omega
 
+theorem toInt_neg_eq_ite {x : BitVec w} :
```

Comment:
```suggestion
theorem toInt_neg {x : BitVec w} :
```
I think the short name is fine for the "no side conditions" statement, unless you think we will need other lemmas with the same LHS.

Reply:
Yeah, I'm uncertain here. The `bmod` version is nice if you're already in a context where you're reasoning about `bmod`, and it's nice to not have `if` on the right hand side. Perhaps neither of them get the short name?

I'm happy for you to make the call here, and we can merge either way.

Reply:
I elected to give neither of them a short name.